snapcraft.yaml

```diff
@@ -0,0 +1,22 @@
+name: errbot
+version: master
+summary: chatbot that brings your tools into the conversation.
+description: |
+  Errbot is a chatbot. It allows you to start scripts interactively from your
+  chatrooms for any reason: random humour, chatops, starting a build, monitoring
+  commits, triggering alerts...
+
+grade: devel
+confinement: strict
+
+apps:
+  errbot:
+    command: env LC_ALL=C.UTF-8 errbot
+    plugs: [home, network]
+
+parts:
+  errbot:
+    source: .
+    plugin: python
+    build-packages: [libffi-dev, libssl-dev]
+    stage-packages: [git]
```

Comment:
ok, I can add them to this PR.
I personally prefer to do that as a separate PR, to better link each change with a specific requirement. This change works for a basic bot, I find it nice to keep track of which additions are needed for more complex one.

But, as I said, just let me know and I'll make the change :)